integration/test-integration/src/test/resources/configs/jwt-generator-config.toml

```diff
@@ -11,6 +11,11 @@
   maxRouteTimeoutInSeconds = 60
   routeIdleTimeoutInSeconds = 300
 
+[router.ratelimit]
+  enabled = true
+  hostname = "172.17.0.1"
```

Comment:
What is this IP? Can't we use 'ratelimit' which is the service name in the docker-compose?